source/FreeRTOS_RA.c

```diff
@@ -295,11 +295,11 @@
                     /* coverity[misra_c_2012_rule_11_3_violation] */
                     pxPrefixOption = ( ( ICMPPrefixOption_IPv6_t * ) &( pucBytes[ uxIndex ] ) );
 
-                    FreeRTOS_printf( ( "RA: Prefix len %d Life %lu, %lu (%pip)\n",
+                    FreeRTOS_printf( ( "RA: Prefix len %d Life %u, %u (%pip)\n",
                                        pxPrefixOption->ucPrefixLength,
                                        FreeRTOS_ntohl( pxPrefixOption->ulValidLifeTime ),
                                        FreeRTOS_ntohl( pxPrefixOption->ulPreferredLifeTime ),
-                                       pxPrefixOption->ucPrefix ) );
+                                       ( void * ) pxPrefixOption->ucPrefix ) );
```

Comment:
Did you get a complaint about an automatic/implicit cast from `uint8_t *` to `void *` ? Which program?